flashpay/apps/account/models.py

```diff
@@ -23,7 +27,7 @@ def __str__(self) -> str:
 
 class APIKey(BaseModel):
 
-    account = models.OneToOneField(Account, on_delete=models.CASCADE, related_name="api_keys")
+    account = models.ForeignKey(Account, on_delete=models.CASCADE, related_name="api_keys")
```

Comment:
Maybe add a comment that it can only two and that's whether the platform is on testnet or mainnet